automation/test.sh

```diff
@@ -27,6 +27,18 @@ readonly TEMPLATES_SERVER="gs://kubevirt-vm-images"
 readonly BAZEL_CACHE="${BAZEL_CACHE:-http://bazel-cache.kubevirt-prow.svc.cluster.local:8080/kubevirt.io/kubevirt}"
 
 
+# Skip if it's docs changes only
+# Only if we are in CI, and this is a non-batch change
+if [[ ${CI} == "true" && -n "$PULL_BASE_SHA" && -n "$PULL_PULL_SHA" ]]; then
+    SKIP_PATTERN="^((docs)/.*|.*/(OWNERS|OWNERS_ALIASES|\.(md|txt)))$"
```

Comment:
So I checked the script with a change in `README.md` which it didn't catch. So I believe that the original expression only works for all OWNERS/ALIASES, txt and md files in subdirectories, but not for the root level.

```suggestion
    SKIP_PATTERN="^((docs)/.*|.*(OWNERS|OWNERS_ALIASES|.*\.(md|txt)))$"
```

Reply:
This one is not perfectly valid either TBH, but I just wanted to point out what I meant.

Reply:
That might be better:

```suggestion
    SKIP_PATTERN="^((docs)/.*|(.*/)*(OWNERS|OWNERS_ALIASES|.*\.(md|txt)))$"
```

Reply:
nice
we can also take `basename` and then we just mess with the filename wdyt?

Reply:
I've reworked the expression.